### ISSUE TITLE
Quality of Life requests

### ISSUE THREAD

**Issue by GoodOleJar** [Crendgrim/AutoHUD#118]
Allow for multiple conditions to trigger one event (IE Health will be revealed when changing OR when low)
Allow for one condition to trigger multiple events (IE When swapping items in hotbar all components will be revealed instead of just the hotbar)

**Comment by Crendgrim** [comment]
I agree. This is all planned with a big rewrite of the mod. I didn't have enough time to finish it in time for 1.21, though. Certainly on my planned features list.

**Comment by GoodOleJar** [comment]
> I agree. This is all planned with a big rewrite of the mod. I didn't have enough time to finish it in time for 1.21, though. Certainly on my planned features list.

Definitely looking forward to these new features.

**Comment by Crendgrim** [comment]
Took quite long, but it's finally in AutoHUD 9.
